### DISCUSSION TITLE
Apple Codesign 0.19.0

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/apple-platform-rs#21] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/indygreg/apple-platform-rs/releases/tag/apple-codesign/0.19.0'>Apple Codesign 0.19.0</a>.</em>

**Comment by jansorg** [comment]
Just discovered rcodesign, which really looks promising!
I was using a setup of osxcross to cross-compile and https://github.com/mitchellh/gon to sign and notarize (on a macOS machine).
This project looks like a suitable replacement - thank you for your hard work on it!
